test/JDBC/input/dml/babel_delete.sql

```diff
@@ -94,9 +94,113 @@ delete babel_2020_delete_t1 from babel_2020_delete_t1 x left outer join babel_20
 go
 
 -- will be tracked in BABEL-3910
---exec babel_2020_delete_ct;
---delete babel_2020_delete_t1 from babel_2020_delete_t2 left outer join babel_2020_delete_t1 x on babel_2020_delete_t2.a = x.a;
---go
+drop procedure if exists babel_3910_init_tables
+go
+
+create procedure babel_3910_init_tables as
+begin
+    drop table if exists t1_3910
+    create table t1_3910 (a int)
+    insert into t1_3910 values (1), (2), (3), (4), (NULL)
+    drop table if exists t2_3910
+    create table t2_3910 (a int)
+    insert into t2_3910 values (2), (3), (4), (5), (NULL)
+    drop table if exists t3_3910
+    create table t3_3910 (a int)
+    insert into t3_3910 values (3), (4), (5), (6), (NULL)
+    drop table if exists t4_3910
+    create table t4_3910 (a int)
+    insert into t4_3910 values (4), (5), (6), (7), (NULL)
+end
+go
+
+exec babel_3910_init_tables;
+delete t1_3910 from t2_3910 left outer join t1_3910 on t2_3910.a = t1_3910.a;
+go
+
+select a from t1_3910;
```

Comment:
there is also a wired way of using join like this  (comma separated list that indicating inner join) 

update t3 set ... from T1, T2 left outer join t3 on ... where t1.a = t2.a